dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -196,4 +140,65 @@ private string DebuggerDisplay
             return display;
         }
     }
+
+    #region Error handling
+    /// <summary>
+    /// Whether an error occurred while executing functions in the pipeline.
+    /// </summary>
+    public bool ErrorOccurred { get; private set; }
```

Comment:
Would remove completely the ErrorOccurred, LastErrorDescription and Leave only the `LastException`, if `LastException` is not null (Alone this means an ErrorOccurred) + LastErrorDescription = (`LastException.Message`)

Reply:
Yup, agreed. (We were also discussing in another related PR). For this change, I'm trying to keep it focused to just the indexer[] and leave the rest of the API as intact as possible. Going forward, I expect ALL of these exception fields will be removed from the SKContext. Instead, they'll either throw directly (critical exceptions) or, if we do want to capture error conditions in the functions, the LastException (plus Result/ModelResults) should be moved to a new `SKReturn` class.